nav2_theta_star_planner/include/nav2_theta_star_planner/theta_star.hpp

```diff
@@ -185,7 +189,10 @@ class ThetaStar
   bool isSafe(const int & cx, const int & cy, double & cost) const
   {
     double curr_cost = getCost(cx, cy);
-    if (curr_cost < LETHAL_COST) {
+    if ((costmap_->getCost(cx, cy) == UNKNOWN_COST && allow_unknown_) || curr_cost < LETHAL_COST) {
+      if(costmap_->getCost(cx, cy) == UNKNOWN_COST) {
```

Comment:
Linting issue, which I'm sure CI picked up on

Reply:
CI actually shows 2 linting issues

Reply:
Can you recommend a tool for autostyling the code so it complies with the project's standards? or do you have a clang-format file I can use?

Reply:
ament_cpplint and ament_uncrustify or just run colcon test locally which will invoke them